confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -81,6 +81,13 @@ target_config_nodes=0.0.0.0:22277
 # Datatype: Double
 # data_region_per_processor=0.5
 
+# Region allocate strategy
+# These allocate strategies are currently supported:
+# 1. GREEDY(Default, when region is allocated, always choose the dataNode that has bean allocated the least regions)
+# 2. COPY_SET(Random replication according to wight calculated from number of regions on all online dataNodes, suitable for large clusters)
+# Datatype: String
+# region_allocate_strategy=GREEDY
```

Comment:
When we add new parameters, please update the 
docs/UserGuide/Reference/ConfigNode-Config-Manual
docs/zh/UserGuide/Reference/ConfigNode-Config-Manual

Reply:
done